.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         go install ./tools/build_gcsfuse
         build_gcsfuse . /tmp ${GITHUB_SHA}
     - name: Test
-      run: go test -p 1 -count 1 -v -cover ./...
+      run: go test -p 1 -count 1 -v -cover ./... $(go list ./... | grep -v /tools/integration_tests/)
```

Comment:
can you add comments what this command does.

Reply:
fixed

Reply:
did you look at this: https://stackoverflow.com/questions/25965584/separating-unit-tests-and-integration-tests-in-go

Reply:
Introduce  --integrationTest flag to run the tests.